agents/postgres/pgstatstatements/pgstatstatements_test.go

```diff
@@ -322,7 +322,7 @@ func TestPGStatStatementsQAN(t *testing.T) {
 			Common: &agentpb.MetricsBucket_Common{
 				Fingerprint:         selectAllCitiesLong,
 				Database:            "pmm-agent",
-				Tables:              []string{"city"},
+				Tables:              []string{},
```

Comment:
then it's wrong test `agentpb.MetricsBucket_Common` should contain table name

Reply:
It is part of the [AllCitiesTruncated](https://github.com/percona/pmm-agent/pull/258/files#diff-aa91ff13c274f4a1ba9eff1e32dd8bd3f4a8566b6bf4d1135408112bcf34d892R255) and the extract tables function fails since the query is not valid.
The logger shows `Can't extract table names from query SELECT /* AllCitiesTruncated:pgstatstatements */ * FROM city WHERE`
The [AllCities](https://github.com/percona/pmm-agent/pull/258/files#diff-aa91ff13c274f4a1ba9eff1e32dd8bd3f4a8566b6bf4d1135408112bcf34d892R191) subtest has the list of tables.